devvit.yaml

```diff
@@ -1,2 +1,2 @@
 name: fundraisers-app
-version: 1.1.2.2
```

Comment:
Depends how we want to handle these things: if we want our next released version to be 1.2.0 without dealing with playtest autoincrementing past that during testing, we need to get all our changes onto Main, devvit upload --bump=minor, then it will auto-increment the repo version and we can commit that. Can discuss on discord